### DISCUSSION TITLE
Result pane in SQL Editor has disappeared 😭

### DISCUSSION THREAD

**Discussion by chihatw** [supabase/supabase#20948] [Questions] [answered]
I am a supabase newbie.
The Result pane in the SQL Editor of the Dashboard has disappeared.
I can't see the results of my Query.
How can I display the Result pane? 😭

**Comment by GaryAustin1** [comment, marked as answer]
Probably need more info.  Try hard refreshing your browser and a different browser.
It is working for me in Firefox.
![image](https://github.com/supabase/supabase/assets/54564956/3263fab0-8aeb-4986-926e-4a7ecd7ec036)

**Comment by chihatw** [comment]
Thanks for your advice.
The Result pane is visible in Firefox.
In my browser, once I clear the local storage, The result pane is back!!
![pic](https://github.com/supabase/supabase/assets/10196689/e3a95ca3-79a8-462f-b3a1-470dd536310a)
